src/core/src/runtime/ov_tensor.cpp

```diff
@@ -113,6 +126,123 @@ Shape Tensor::get_shape() const {
     OV_TENSOR_STATEMENT({ return _impl->getTensorDesc().getBlockingDesc().getBlockDims(); });
 }
 
+void Tensor::copy_to(ov::Tensor& dst) const {
+    OPENVINO_ASSERT(dst.get_element_type() == get_element_type(),
```

Comment:
how does it work for:
- RemoteTensor::copy_to(Tensor)
- RemoteTensor::copy_to(RemoteTensor)
- Tensor::copy_to(RemoteTensor)

Reply:
I will add a check that tensors shouldn't be remote